### PR TITLE
show correct age & display time to next birthday

### PR DESCRIPTION
**Pull request by thomd**
The "age" column does now show the correct age if it is a forthcoming birthday. I also changed the right column to display the time to the next birthday (also if it is in the next year).

### CODE REVIEW

**Comment by IonicaBizau** [comment]
Thanks @thomd! :sparkles: 

> The "age" column does now show the correct age if it is a forthcoming birthday.

Can you please give me an example where the current implementation fails? I cannot reproduce it.

File: bin/birthday [hunk 1]
```diff
@@ -6,7 +6,7 @@ var Birthday = require("../lib")
   , Package = require("../package")
   , Logger = require("bug-killer")
   , Table = require("le-table")
-  , Moment = require("moment")
+  , moment = require("moment")
```
Comment (IonicaBizau) [inline bin/birthday]: Please keep it with uppercase, that's how I `npm`. :smile:

**Comment by thomd** [comment]
if I add a birthday of 01.Oct and 31.Oct, then both have the same age as of today (which is 10.Oct). 

``` bash
birthday [master] > birthday -n "foo" -d "01/10/2000"
info  Inserted successfully

birthday [master] > birthday -n "bar" -d "31/10/2000"
info  Inserted successfully

birthday [master] > birthday
┌─┬────┬──────────┬──────────┬───────────┐
│#│Name│Birthday  │Age       │           │
├─┼────┼──────────┼──────────┼───────────┤
│1│foo │01 October│15        │10 days ago│
├─┼────┼──────────┼──────────┼───────────┤
│2│bar │31 October│15        │in 20 days │
└─┴────┴──────────┴──────────┴───────────┘
```

I'd like to have them showing their age of today: "bar" is 14 and "foo" is 15.

**Comment by IonicaBizau** [comment]
Ah, got it! The current implementation only shows the age the person has after his birthday.
But your idea is good. I will test it out.

In the meantime, please add yourself as contributor in `package.json`.

**Comment by IonicaBizau** [comment]
Hey @thomd! :star: 

I checked your changes. I'm not so happy with _in a year_ snippets. It may happen that I forgot to tell _Happy Birthday_ but I see it next day and it's better to see there _a day ago_ instead of _next year_.

And for the age column, is it possible to show the age and the age after birthday too?

Something like this:

```
18 → 19
```

That means the person turns 19, but currently they are 18.

Thanks! :dizzy:

**Comment by IonicaBizau** [comment]
@thomd Sorry it took so long to merge this. I pushed some fixes and :ship: **`1.1.0`**. :tada:

**Pull request merged**